Build/Projects/MonoGame.Framework.definition

```diff
@@ -292,7 +292,7 @@
     <Compile Include="DisplayOrientation.cs" />
     <Compile Include="DrawableGameComponent.cs" />
     <Compile Include="FrameworkDispatcher.cs">
-      <Platforms>_FrameworkDispatcherProvided</Platforms>
+      <Services>_FrameworkDispatcherProvided</Services>
```

Comment:
Yeah, I know that's what's supposed to happen, but that wil reintroduce the stutter :/ If we really wanna fix this we'll have to dig into SharpDX code I think. Let me just try this locally to make sure